addons/sale/controllers/portal.py

```diff
@@ -166,6 +166,9 @@ def portal_order_page(
             'report_type': 'html',
             'backend_url': backend_url,
             'res_company': order_sudo.company_id,  # Used to display correct company logo
+            'is_payment_provider_active': request.env['payment.provider'].sudo().search([
+                ('state', 'in', ['test', 'enabled'])
```

Comment:
```suggestion
                ('state', 'in', ['test', 'enabled']),
```
Also need to check company of the provider :eyes: